### PR TITLE
Change tests to dump the first migration and use for all other tests

### PR DESCRIPTION
**Pull request by diversemix**
# Description

- **add default env vars for pg client**
- **add postgresql-client to the docker image**
- **Change tests to dump the first migration and use for all other tests**

Please include:

- a summary of the changes
- links to any related issue/ticket
- any additional relevant motivation and context
- details of any dependency updates that are required for this change

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] **DB_CLIENT DEPENDENCY IS ON THE LATEST VERSION**
- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: tests/integration_tests/conftest.py [hunk 1]
```diff
@@ -40,29 +39,51 @@
...
+    else:
+        result = subprocess.run(["psql", "-f", migration_file])
+        assert result.returncode == 0
+        test_engine = create_engine(test_db_url)
+
+    return test_engine
+
+
+@pytest.fixture(scope="function")
+def slow_db(monkeypatch):
```
Comment (katybaulch) [inline tests/integration_tests/conftest.py:76]: A bit pedantic, but shouldn't this now be a quick_db rather than a slow DB if the migrations are cached?

**Review by katybaulch — DISMISSED** [review]
All of the cookies, Thankyou so much!!!🍪⭐

File: tests/integration_tests/conftest.py (hunk 1 above)
Comment (diversemix) [reply tests/integration_tests/conftest.py:76]: Ah fair point ... I can probably remove that  now tbf :eye:

**Review by katybaulch — APPROVED** [review]
Amazing!!!!!

**Pull request merged**